CHANGELOG_PENDING.md

```diff
@@ -16,10 +16,14 @@
 
 #### General
 
+- [#1813](https://github.com/livepeer/go-livepeer/pull/1813) Check that priceInfo.pixelsPerUnit is not 0
```

Comment:
We don't need the link here since it will be automatically added when a release is cut so let's use the format described in the [contribution](https://github.com/livepeer/go-livepeer/blob/master/doc/contributing.md#changelog) guide which will allow that link to be automatically added later on.